### PR TITLE
Implemented DB

### PR DESCRIPTION
**Pull request by BFARIAAL**
Implemented single Database and changed application properties to allow for integration with frontend

### CODE REVIEW

File: src/main/resources/application.properties
Comment (Nick7OlFox) [inline src/main/resources/application.properties:1]: Why did you delete the file

**Review by Nick7OlFox — CHANGES_REQUESTED** [review]
Please resolve comments

File: src/main/resources/application.properties
Comment (BFARIAAL) [reply src/main/resources/application.properties:1]: Changed it to application.yml

**Review by Nick7OlFox — APPROVED** [review]
